src/sage/modular/cusps_nf.py

```diff
@@ -901,7 +901,7 @@ def ABmatrix(self):
 
         return ABM
 
-    def is_Gamma0_equivalent(self, other, N, Transformation=False):
+    def is_Gamma0_equivalent(self, other, N, Transformation=False) -> bool:
```

Comment:
```suggestion
    def is_Gamma0_equivalent(self, other, N, Transformation=False) -> bool | tuple[bool, Any]:
```
and import `typing.Any`